jadx-core/src/main/java/jadx/api/data/IJavaNodeRef.java

```diff
@@ -2,7 +2,7 @@
 
 public interface IJavaNodeRef extends Comparable<IJavaNodeRef> {
 
-	enum RefType {
+	public enum RefType {
```

Comment:
Not really needed, because all entries inside interface have `pulblic` modifier by default.

Reply:
Removed in f53735209191d912664181e9d5752efcdfbd9c6e!